docs/podman-image-trust.1.md

```diff
@@ -40,32 +34,33 @@ Require signature (“signedBy”).
 
 Trust may be updated using the command **podman image trust set** for an existing trust scope.
 
-# OPTIONS
-**-h** **--help**
+## OPTIONS
+**-h, --help**
   Print usage statement.
 
-**-f** **--pubkeysfile**
+**-f, --pubkeysfile** *KEY1*
   A path to an exported public key on the local system. Key paths
   will be referenced in policy.json. Any path to a file may be used but locating the file in **/etc/pki/containers** is recommended. Options may be used multiple times to
   require an image be signed by multiple keys.  The **--pubkeysfile** option is required for the **signedBy** type.
 
-**-t** **--type**
-  The trust type for this policy entry. Accepted values:
+**-t, --type** *value*
+  The trust type for this policy entry.
+  Accepted values:
     **signedBy** (default): Require signatures with corresponding list of
                         public keys
     **accept**: do not require any signatures for this
             registry scope
     **reject**: do not accept images for this registry scope
 
-# show OPTIONS
+## show OPTIONS
```

Comment:
I know it's preexisting, but should 'show' be 'Show'?

Reply:
Or should it be `show` (with the code markup) , since these options only are for the `show` command?